### ISSUE TITLE
A Couple Powerfist Bugs

### ISSUE THREAD

**Issue by Leanfrog** [ParadiseSS13/Paradise#18959]
**What is the problem**:
1. Placing an air tank on a powerfist while it is in your bag does not apply nodrop to it.
2. You can walk out of the knockback from the punches of the powerfist.

**What did you expect to happen**: 
1. For the powerfist to become nodrop regardless of where it had the tank attached.
2. For the victim to not be able to walk out of the knockback from the punch.

**What happened instead**:
1. The powerfist did not gain nodrop when it had a tank equipped while it was in a bag.
2. The victim was able to walk against the knockback of the powerfist, additionally the victim could even slam themselves into the puncher if they were close enough and timed it right.

**Why is this bad/What are the consequences**:
1. It's odd and can "punish" players for doing something that would be seen as perfectly rational by not giving the nodrop when it should.
2. Knockback should work as intended and not be resistable by just walking, also being able to slam yourself into the attacker shouldn't be a thing.

**Steps to reproduce the problem**:
1. Put a powerfist in your bag, then equip an oxygen tank to it, notice it doesn't have nodrop when removed and you can just toss it around freely.
2. Punch a target with a powerfist that has a filled air tank the target must be capable of moving, have the target walk against the direction of the knockback, notice that they are able to resist the knockback and even slam into the powerfist user if they're directly adjacent.

**When did the problem start happening**:
N/A

**Comment by Christer2222** [comment]
Encountered the tank-in-bag bug myself

**Comment by Spaghetti-bit** [comment]
The tank in bag is not a bug and is intended as far as I’m aware.

**Comment by Christer2222** [comment]
> The tank in bag is not a bug and is intended as far as I’m aware.

I disagree, it feels very awkward that you can put a tank inside the fist while it is in your bag, and then surprise attack someone with it. The powerfist is a very loud and visible weapon, if you want to use it, then it needs to be on your hand. If it really is intended, there should be a line on examine saying something like "putting a gas tank in the powerfist while it is in your bag loosens your grip enough on it to allow you to equip and unequip it at will without the tools needed". Imo stealth attacking with a powerfist might even be considered an exploit.

**Comment by DGamerL** [comment]
Tank in bag is a bug.

**Comment by kyunkyunkyun** [comment]
#23775 and #25274 fixed this i believe
